aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -346,22 +618,22 @@ impl AggregatorData {
         self.aggregators.len() as u128
     }
 
-    /// Creates and a new Aggregator with a given `id` and a `limit`. The value
+    /// Creates and a new Aggregator with a given `id` and a `max_value`. The value
     /// of a new aggregator is always known, therefore it is created in a data
     /// state, with a zero-initialized value.
-    pub fn create_new_aggregator(&mut self, id: AggregatorID, limit: u128) {
+    pub fn create_new_aggregator(&mut self, id: AggregatorID, max_value: u128) {
         let aggregator = Aggregator {
-            value: 0,
-            state: AggregatorState::Data,
-            limit,
-            history: None,
+            id,
+            state: AggregatorState::Data { value: 0 },
+            max_value,
         };
         self.aggregators.insert(id, aggregator);
         self.new_aggregators.insert(id);
     }
 
     /// If aggregator has been used in this transaction, it is removed. Otherwise,
     /// it is marked for deletion.
+    /// TODO: Should we return an error if aggregator id is v2?
```

Comment:
@georgemitenkov where is this method even used? I only see uses in tests? (then we can gate it w. cfg too)